src/tdvp_step.jl

```diff
@@ -209,7 +211,7 @@ function tdvp_site_update!(
     Δ = (isforward(direction) ? +1 : -1)
     orthogonalize!(psi, b + Δ)
   end
-  return current_time, maxtruncerr, spec
+  return current_time, maxtruncerr, spec, converged
```

Comment:
As stated above, I think we should return the entire `info` object here, not just the `info.converged` field.

Reply:
I have done as you asked.